test/FunctionalTests/Microsoft.OData.Core.Tests/Json/ODataJsonWriterUtilsAsyncTests.cs

```diff
@@ -21,58 +21,15 @@ public class ODataJsonWriterUtilsAsyncTests
     {
         private StringWriter stringWriter;
         private IJsonWriterAsync jsonWriter;
-        private ODataMessageWriterSettings settings;
         private Func<ICollection<ODataInstanceAnnotation>, Task> writeInstanceAnnotationsDelegate;
 
         public ODataJsonWriterUtilsAsyncTests()
         {
             this.stringWriter = new StringWriter();
             this.jsonWriter = new JsonWriter(this.stringWriter, isIeee754Compatible: true);
-            this.settings = new ODataMessageWriterSettings();
             this.writeInstanceAnnotationsDelegate = async (ICollection<ODataInstanceAnnotation> instanceAnnotations) => await TaskUtils.CompletedTask;
         }
 
-        [Fact]

```

Comment:
Do we have a test elsewhere that shows that jsonp is still supported? i.e. that if there's jsonp callback set, that the output will be wrapped in a json function?

Reply:
@habbes Yes we do. After removing the async methods from `ODataJsonWriterUtils`, I confirmed that the code I transferred to `ODataJsonLightSerializer` is still being tested: https://github.com/OData/odata.net/blob/57572612f1ff833d395b44e6b2c498d7d0d31890/test/FunctionalTests/Microsoft.OData.Core.Tests/JsonLight/ODataJsonLightSerializerTests.cs#L306-L326